lib/thunderbird_common.pm

```diff
@@ -73,16 +73,17 @@ sub tb_setup_account {
                 send_key 'down';
                 send_key 'ret';
             }
-            assert_screen "thunderbird_wizard-$proto-selected";
+            if (is_sle) {
```

Comment:
How can this be a SLE specific behavior? isn't it rather depending on version only?

Reply:
Ok, I think now is better. Thanks